src/vpn/dnsIsAlive.go

```diff
@@ -13,11 +13,12 @@ func (h *Handler) dnsIsAlive() (bool, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 	defer cancel()
 
-	if !nettools.HasIPv6PingCommand() {
+	if nettools.HasIPv6PingCommand() {
```

Comment:
I don't think the removal of `!` is correct.

Reply:
Yeah, my bad